third_party/yarr/Vector.h

```diff
@@ -210,6 +210,187 @@ class Vector {
 
 };
 
+template <typename T>
+class GCVector {
+public:
+    using iterator = T*;
+    using const_iterator = const T*;
+    Escargot::Vector<T, GCUtil::gc_malloc_allocator<T>> impl;
```

Comment:
minor) what about defining `impl` as a private member, not public?

Reply:
Good idea!

Reply:
I've updated pr